sdk/storage/perf-tests/storage-blob/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "../../../../tsconfig.package",
+  "compilerOptions": {
+    "target": "es5",
+    "module": "commonjs",
+    "declarationDir": "./typings/latest",
+    "outDir": "./dist-esm",
+    "lib": ["dom", "es5", "es6", "es7", "esnext"]
+  },
+  "compileOnSave": true,
```

Comment:
does this work with rush?

Reply:
Same, copied the tsconfig for `storage-blob` SDK.

Reply:
compileOnSave is working when tried editing TS files in VSCode.